app/code/Magento/Catalog/view/adminhtml/web/catalog/product-attributes.js

```diff
@@ -26,15 +26,15 @@ define([
         _initModal: function () {
             var self = this;
 
-            this.modal = $('<div id="create_new_attribute"/>').modal({
+            this.modal = $('#create_new_attribute').modal({
                 title: $.mage.__('New Attribute'),
                 type: 'slide',
                 buttons: [],
 
                 /** @inheritdoc */
                 opened: function () {
                     $(this).parent().addClass('modal-content-new-attribute');
-                    self.iframe = $('<iframe id="create_new_attribute_container">').attr({
+                    self.iframe = $('#create_new_attribute_container').attr({
```

Comment:
It looks like the iframe element is not referenced but created here, so I don't think this change is correct